discovery-provider/src/queries/queries.py

```diff
@@ -1982,3 +1983,130 @@ def get_top_genre_users():
 
 
         return api_helpers.success_response({ 'user_ids': user_ids })
+
+
+
+# Get the tracks that are 'children' remixes of the requested track
+# The results are sorted by if the original artist has reposted or saved the track
+@bp.route("/remixes/<int:track_id>/children", methods=("GET",))
+def get_remix_tracks(track_id):
+
+    db = get_db_read_replica()
+    with db.scoped_session() as session:
+        parent_track = session.query(Track).filter(
```

Comment:
this is a huge select, it's selecting every column from Track, Remix, Save and Repost. If we don't need all these columns we should trim what we select

Reply:
Was this comment meant for this line? This is just selecting the 1 track by track id? Really thought, I could only query for the `owner_id` column, but is that what you mean?

Reply:
@jowlee sorry,  meant to comment this in line 2005

Reply:
I believe it's not returning every column from the tables Track, Remix, Save, Repost, just the columns from the Track table right?